etc/cmake/options.cmake

```diff
@@ -82,6 +82,11 @@ if(OT_BORDER_ROUTING)
     target_compile_definitions(ot-config INTERFACE "OPENTHREAD_CONFIG_BORDER_ROUTING_ENABLE=1")
 endif()
 
+option(OT_BORDER_ROUTING_NAT64 "enable border routing NAT64 support")
+if(OT_BORDER_ROUTING_NAT64)
+    target_compile_definitions(ot-config INTERFACE "OPENTHREAD_CONFIG_BORDER_ROUTING_NAT64_ENABLE=1")
```

Comment:
Should we also add a similar option in `examples/common-switches.mk`?

Also adding documentation for the new options in `examples/README.md`?

Reply:
Updated